src/profile-logic/marker-timing.js

```diff
@@ -155,6 +162,17 @@ export function getMarkerTimingAndBuckets(
       // Keep the original ordering if the names are the same.
       return 0;
     }
+
+    // Put network requests at the end of the Network category.
+    if (a.bucket === 'Network') {
+      if (a.name === 'Network Requests') {
```

Comment:
I don't really like that the name is hard coded in 2 places, but the tests should prevent this from regressing. But happy to extract to a constant if you prefer.

Reply:
Yeah, it's not so great but we have the similar checks above already. So it's works for me either way.